src/im_tables/views/dashboard/save.cljs

```diff
@@ -43,7 +44,10 @@
                     ; Close the modal by clearing the modal markup in app-db
                     (dispatch [:prep-modal loc nil]))]
 
-    {:header [:h4 (str "Save a list of " (:count details) " " (if (< count 2) (name type) (plural (name type))))]
+    {:header [:h4 (str "Save a list of " (:count details) " "
+                       (if (and (number? count) (< count 2))
```

Comment:
What scenario might the count not be a number?

Reply:
While it's loading it uses the placeholder `...`.

Reply:
could we add that in as a comment?

Reply:
Sure